api/seeds/shared/set-up-users.js

```diff
@@ -89,17 +88,6 @@ const createUsersToAdd = async (knex, oktaClient) => {
       status: 'approved',
       username: 'em@il.com'
     });
-    // Add an expired certification and this user will be downgraded to "regular user"
```

Comment:
are you going to seed this data anywhere? or do you want to add it in a future ticket when we need some examples of matched certifications and users.

Reply:
So we previously had certifications attached to 2 non-eAPD State Admin users. I removed those as they didn't make sense. I removed the `state_admin_certifications_audit` (and related seed data) based on our dev sync. Finally, I updated the certification that is attached to our eAPD State Admin user. So this should be correct given where we are at in the fed admin feature implementation.